README.md

```diff
@@ -0,0 +1,51 @@
+# ARSIS-6
+The sixth iteration of of the ARSIS project.
+
+## Project Summary
+
+The Boise State NASA SUITS team is currently in the process of developing an augmented 
+reality application, Augmented Reality Space Informatics System (ARSIS). The purpose of 
+ARSIS is to help astronauts more easily complete EVA tasks.
+
+Currently, real-life astronauts’ mission objectives are completed via written or 
+memorized instructions. SUITS addresses this problem by creating an AR environment which 
+will also communicate with ground control, and then be displayed in the astronaut’s helmet 
+with up-to-date information and instructions about the task at hand. 
+
+See last year's [ARSIS project](https://github.com/NASA-SUITS-Teams/ARSIS-6).
```

Comment:
We should double check this is correct @Aetheris743  I think should know but if memory serves this has a weird history of why its laid out like this